sdk/keyvault/keyvault-admin/package.json

```diff
@@ -47,13 +47,13 @@
     "build:nodebrowser": "rollup -c 2>&1",
     "build:test": "echo skipped",
     "build": "npm run extract-api && npm run build:es6 && npm run build:nodebrowser",
-    "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
```

Comment:
should we add `--ignore-path ../../../.prettierignore` here too for future-proofing?  The npm scripts are also more consistent with other libs.

Reply:
I am thinking of ditching those hard coded paths because `prettier` is already smart enough to find the config files automatically, [see here](https://github.com/Azure/azure-sdk-for-js/pull/10416#discussion_r464625899). Is there a specific reason for hardcoding that I am not aware of?

Reply:
I am happy as long as they are consistent.

Reply:
I commented in another thread, but I agree the hard coded paths should be removed if the commands work correctly without them.

Reply:
> should we add --ignore-path ../../../.prettierignore here too for future-proofing?

done.